m/method_source/method_source_ubi_8.5.sh

```diff
@@ -1,42 +1,74 @@
 #!/bin/bash -e
 # ----------------------------------------------------------------------------
 #
-# Package       : method_source
-# Version       : 0.9.2
-# Source repo   : https://github.com/banister/method_source
-# Tested on     : UBI: 8.5
-# Language      : Ruby
-# Travis-Check  : True
-# Script License: Apache License 2.0
-# Maintainer's  : Balavva Mirji <Balavva.Mirji@ibm.com>
```

Comment:
add to the maintener field, don't remove the name of current maintainer

Reply:
I and Balavva raised PR on the same day..there was a confict that i copied her script and added to my PR accidently and then i deleted it..her PR got merged..she worked on 0.9.2 ..where i have raised PR for top of tree